### DISCUSSION TITLE
Products categories for PoS (Folder of Product and VAT calculation)

### DISCUSSION THREAD

**Discussion by yannick10800** [btcpayserver/btcpayserver#2624] [Ideas / Feature Requests]
**Problem**
In case of multiple products, there is a list of producs for shop and there is no way to sort or to put in folder.
There is also no VAT calculation
**Solution**
For easier use , some clients would require a folder of products (e.g. restaurants DRINKS: Beer, Cola, Wine ... ICECREAM A, B, C)

**Examples**
DRINKS Beer, Cola, Wine
**Additional context**
Example of possible folder management 

[https://lh3.googleusercontent.com/vUbp0sciMjkhQ50pVeJngLQsU4e3V-6KmUYDz8JSm4XCRAD6Kf2Xmhy22MGEAmbr1A=w1920-h941-rw](url)

**Comment by pavlenex** [comment]
Can you elaborate a bit? It's not quite clear what products are you referring to, is it in the invoice, or the point of Sale App? Please update your issue with more information.

**Comment by yannick10800** [comment]
### For Point of Sale App
For more convinient use for the end user (shop), I would need to have the possibilty to have a Folder and insinde the folder, some producs
example : 
DRINKS(folder) Beer, tea, coffee (Products)
FOOD (folder)  Producs

### For the invoice
The calculation of VAT is missing

**Comment by yannick10800** [comment]
@pavlenex Do you have enough information?

**Comment by pavlenex** [comment]
From what I understand, you want to have categories for Point of Sale app?
Drinks > Beer > Cold Beer > Black Cold Beer?

The vat feature you asked is related to #614

**Comment by yannick10800** [comment]
Yes Categories for Point of Sale app
